src/main/java/io/github/dftrakesh/cloverrest/model/Order/OrderElements.java

```diff
@@ -0,0 +1,11 @@
+package io.github.dftrakesh.cloverrest.model.Order;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OrderElements {
+    private List<OrderElement> elements;
```

Comment:
change name elements to orderElementList

Reply:
Changed name elements to orderElementList